build-ps/debian/libperconaserverclient18.2.docs

```diff
@@ -0,0 +1,2 @@
+CMakeLists.txt
```

Comment:
Evgeniy @efirs: do we need to pack CMakeLists.txt? for whom?

Reply:
@abychko as I understand it is just file renaming from libperconaserverclient18.1.docs to libperconaserverclient18.2.docs. In our current file we have CMakeLists.txt included.